frontend/src/pages/LogInPage/LogInPage.css

```
@@ -0,0 +1,70 @@
+
+.auth-button{
+    margin-top: 10px;
+    margin-right: 10px;
+}
+
+.email{
+    margin-top: 10px;
+    display:flex;
```

```suggestion
.email {
    margin-top: 10px;
    display: flex;
```
add spaces, also apply to others